cmd/scaffold.go

```diff
@@ -671,11 +671,11 @@ func (fnb *FlowNodeBuilder) initFvmOptions() {
 		fvm.WithChain(fnb.RootChainID.Chain()),
 		fvm.WithBlocks(blockFinder),
 		fvm.WithAccountStorageLimit(true),
+		fvm.WithTransactionFeesEnabled(true),
```

Comment:
what is the impact of this on localnet and benchnet, it might break loaders, was wondering if we could still filter it by chainID be mainnet,

Reply:
I think we've used the loaders on canary before no? And i believe fees are enabled there. 

Or did we do extra setup to get the loaders to work for canary?

Reply:
you mean the marketplace simulator? I was talking about the default loader that we have for localnet, not sure if we have run it against canary before

Reply:
We ran in on both testnet and canary chain before... do the localnet and benchnet have a separate chain id? It might be better to exclude those then.